text/0000-lock-without-registry.md

```diff
@@ -0,0 +1,35 @@
+- Start Date: 2017-05-09
+- RFC PR:
+- Yarn Issue:
+
+# Summary
+
+The lockfile yarn.lock should not include the base registry (`https://registry.npmjs.org`).
+
+# Motivation
+
+In yarn.lock, the `resolved` field includes registry such as `https://registry.npmjs.org`.
+In China, most developers will set it to `https://registry.npm.taobao.org` for speed; but it seems slow for travis-ci and circleci.
+
+# Detailed design
+
+Replace the `resolved` by a `hash` field.
```

Comment:
@bestander as noted in my motivation comment, retaining the registry URL within in a `yarn.lock` file could leak company repositories, so entirely removing them from a `yarn.lock` file would be nice.

> Actually it would require everyone to update their lockfiles to have the partial tarballs URLs, that may limit the usefulness of this feature.

I can see that it would require intervention by each project to upgrade their `yarn.lock` files to follow the new approach.